src/style-spec/reference/v8.json

```diff
@@ -2524,6 +2500,72 @@
     "value": "*",
     "doc": "A filter selects specific features from a layer."
   },
+  "filter_symbol": {
```

Comment:
Is there logic to whether these are snake_cased or camelCased? I definitely see both in the style-spec. Maybe it's that root properties are snake_cased and nested properties are camelCased?

Reply:
In general its reverse snake case to denote nesting. like `layout_symbol` is used to denote the structure of `symbol.layout` object.
Hence the similar notation for this, `filter_symbol` denotes the structure of the `symbol.filter` object.

Reply:
Thanks for the clarification!